### PR TITLE
HTML-707 Update ConditionTagHandlerSupport2_2 to work for later versions

### PR DESCRIPTION
**Pull request by wluyima**
## Issue I worked on

see https://issues.openmrs.org/browse/HTML-707

## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api-2.2/src/main/java/org/openmrs/module/htmlformentry/handler/ConditionTagHandlerSupport2_2.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import org.openmrs.module.htmlformentry.FormEntrySession;
 import org.openmrs.module.htmlformentry.FormSubmissionController;
 
-@OpenmrsProfile(openmrsPlatformVersion = "2.2.*")
+@OpenmrsProfile(openmrsPlatformVersion = "2.2.0")
```
Comment (mogoodrich) [inline api-2.2/src/main/java/org/openmrs/module/htmlformentry/handler/ConditionTagHandlerSupport2_2.java]: Why does this get changed to *only* be loaded for 2.2.0?

Comment (mks-d) [reply api-2.2/src/main/java/org/openmrs/module/htmlformentry/handler/ConditionTagHandlerSupport2_2.java]: @mogoodrich it's counter intuitive but this syntax actually means "2.2.0 and all version above".

Comment (mogoodrich) [reply api-2.2/src/main/java/org/openmrs/module/htmlformentry/handler/ConditionTagHandlerSupport2_2.java]: Yeah, just saw your comment on the ticket... I flagged Daniel, I'm curious if this is intended behavior...

Comment (mks-d) [reply api-2.2/src/main/java/org/openmrs/module/htmlformentry/handler/ConditionTagHandlerSupport2_2.java]: I was surprised as well, it's actually "documented" here: https://github.com/openmrs/openmrs-core/blob/a5c3bfc890132b3f50eb4febba5e1237a165778d/api/src/main/java/org/openmrs/module/ModuleUtil.java#L258

Comment (mogoodrich) [reply api-2.2/src/main/java/org/openmrs/module/htmlformentry/handler/ConditionTagHandlerSupport2_2.java]: Yeah, I'm not crazy about it, but it is intentional from that documentation...

**Pull request merged**